src/main.rs

```diff
@@ -0,0 +1,83 @@
+use mdbook::renderer::RenderContext;
+use serde::Deserialize;
+use std::collections::BTreeMap;
+use std::io;
+
+#[derive(Deserialize)]
+struct I18nConfiguration {
+    languages: BTreeMap<String, String>,
+    default_language: Option<String>,
+
+    #[serde(default)]
+    translate_all_languages: bool,
+    #[serde(default)]
+    move_translations_to_html_directory: bool,
+}
+
+fn main() {
+    let mut stdin = io::stdin();
+
+    // Get the configs
+    let ctx = RenderContext::from_json(&mut stdin).unwrap();
+    let i18n_config: I18nConfiguration = ctx
+        .config
+        .get_deserialized_opt("output.i18n-helpers")
+        .unwrap()
+        .unwrap();
```

Comment:
Not really your fault... but how does the error look if someone enables the renderer but doesn't fill out any of the values (I guess `languages` and `default_language` are required values?)

Reply:
Thanks for pointing out, I added `serde(default)` on languages and now, it will just have empty values